test/_init_tests.js

```diff
@@ -33,6 +33,9 @@ import * as describes from '../testing/describes';
 // All exposed describes.
 global.describes = describes;
 
+// Increase the before/after each timeout since certain times they have timedout
+// during the normal 2000 allowance.
+const BEFORE_AFTER_TIMEOUT = 5000;
```

Comment:
If this is only for video tests you should be able to just do it for the video tests, no? Maybe inside the describe of the top level.